ftpcontroller.py

```diff
@@ -55,20 +57,32 @@ def handleDataCommand(self, command, argument, dataSocket):
       return self.readData(command, argument, dataSocket)
     return self.sendData(argument, dataSocket)
 
+  def addBytesToBuffer(self, argument, timeElapsed):
```

Comment:
Code looks good. Could we just move this logic into ftputils and rename it to something like ‘getTransferResponse’